### DISCUSSION TITLE
Running 2 instances of vector under different linux users

### DISCUSSION THREAD

**Discussion by bmallya** [vectordotdev/vector#19322] [General]
Hello,
I have a configuration [v32] where in , there is a first instance which exposes http, file and unix as source and then using vector as a sink forwards it to another instance (Agent - Collector model), where these received events are stored to a file.
Till this point in time, I had them both started by the same linux user and had no issues.
Recently, I had to migrate the collector side to a new user with a larger disk quota associated to it. I am now seeing that most of the times the events/logs are not being forwarded from the agent to the collector. Are there any additional settings on the user that might help? Or are there restrictions preventing it.

Thanks.

**Comment by jszwedko** [comment]
Can you share any error logs from the misbehaving process? There's nothing inherent to Vector that depends on the specific user it is running under but obviously there could be differences on your system around permissions, quotas, etc.

**Comment by bmallya** [comment]
@jszwedko , I am not able to recreate it consistently. Let me keep trying with verbose logs enabled, and will post it the next time I see it.
Thanks
